components/teams-service/server/v1/server.go

```diff
@@ -180,6 +180,10 @@ func (s *Server) UpdateTeam(ctx context.Context, req *teams.UpdateTeamReq) (*tea
 
 // AddUsers associates an array of users with an existing team
 func (s *Server) AddUsers(ctx context.Context, req *teams.AddUsersReq) (*teams.AddUsersResp, error) {
+	if len(req.UserIds) == 0 {
```

Comment:
can't we add this validation in the proto? Or I guess that would break the v1 API